app/config/dependencies.php

```diff
@@ -10,6 +10,7 @@
 require_once __DIR__ . '/../repositories/CommentRepository.php';
 require_once __DIR__ . '/../repositories/UserRepository.php';
 require_once __DIR__ . '/../repositories/BadgeRepository.php';
+require_once __DIR__ . '/../repositories/SocialsRepository.php';
```

Comment:
Can we name it SocialLinkRepsitory ?
beacause the model is SocialLink,
same goes for SocialLinkService & sociallinks.js or social-links.js
🫣

Reply:
yes, However the reason for this naming was the in order to follow the rest of the naming conventions for repositories.

1 word + Repository

Reply:
The rest of it is exactly as the model name and then Repository, service and controller

Reply:
fixed